lib/inspec/runtime_profile.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+# Copyright 2017 Chef Software
+# author: Dominik Richter
+# author: Christoph Hartmann
+
+module Inspec
+  class RuntimeProfile
+    def initialize(profile)
+      org = profile.source_reader.files
+      @files = org.each_with_object({}) do |(k, v), acc|
+        acc[k.sub('files' + File::SEPARATOR, '')] = v
```

Comment:
Do we need to prepopulate this hash with the files AND their contents? The SourceReader is already doing that, and then we do it again here just to have different keys... feels like we're consuming 2x the memory we need for files that may not even get used by the controls.

It feels like we should let the SourceReader provide the list of files and provide a method to return the contents, and all this class has to do is take a parameter, prefix it with "files/" and then pass it to the SourceReader to get us the content... and to list the files, it can strip "files/" before it hands it back to the caller.